storybook/preview.js

```diff
@@ -1,6 +1,5 @@
-import { addParameters, configure } from '@storybook/react';
+import { configure } from '@storybook/react';
```

Comment:
Should you remove the `grommetLight` code if it's not being used?

Reply:
It was removed, wasnt it?